docs/DEVGUIDE.md

```diff
@@ -18,6 +28,26 @@ layout:
     ├── pkg            # Contains all non-"main" Go packages
     └── vendor         # Glide-managed dependencies (untracked)
 
+## Working on Issues
+
+Github does not allow non-maintainers to assign, or be assigned to, issues.
+As such non-maintainers can indicate their desire to work on (own) a particular
+issue by adding a comment to it of the form:
+
+	#dibs
+
+However, it is a good idea to discuss the issue, and your intent to work on it,
+with the other members via the slack channel to make sure there isn't some
+other work alread going on with respect to that issue.
```

Comment:
As a newbie telling me to ping someone (how I know who is a mystery) and ask them to take time from their busy day to ride side-saddle with me on what could be some trivial issue/PR sounds like a roadblock to encouraging new people to join our group.  If we can't agree on something as trivial as a #dibs kind of thing I would prefer to close this PR than add an onerous process.

Keep in mind, people can issue PRs today w/o grabbing issues and w/o talking to an existing maintainer - all we're doing is trying to help them by making sure someone else isn't working on the same issue at the same time - I'm not looking to add hurdles for new PRs.